mainApp.py

```diff
@@ -267,7 +316,7 @@ def showInMap(self, ids, layerName):
                 )
                 return
             else:
-                vectorLayer.setSelectedFeatures(fIds)
+                vectorLayer.selectByIds(fIds)
```

Comment:
@janadamek-programmer Please separate changes related to feature highlighting into new PR (to keep PR problem-specific). It will simplify review process. Thanks.